### PR TITLE
machine-config-daemon: Update help to reference MCD

### PR DESCRIPTION
**Pull request by ashcrow**

### CODE REVIEW

**Comment by sdemos** [comment]
@ashcrow why do the robots hate us?

File: cmd/machine-config-daemon/main.go [hunk 1]
```diff
@@ -14,8 +14,8 @@ const (
 var (
 	rootCmd = &cobra.Command{
 		Use:   componentName,
-		Short: "Run Machine Config Controller",
-		Long:  "",
+		Short: "Run Machine Config Daemon",
+		Long:  "Runs the Machine Config Daemon which handles communication between the host and the cluster",
```
Comment (abhinavdahiya) [inline cmd/machine-config-daemon/main.go]: This should mentions that the daemon applies the machineconfigs to the machine. ?

Comment (ashcrow) [reply cmd/machine-config-daemon/main.go]: I can update it with that.

Comment (ashcrow) [reply cmd/machine-config-daemon/main.go]: Updated

**Pull request merged**